frontend/public/components/operator-lifecycle-manager/install-plan.tsx

```diff
@@ -167,15 +175,20 @@ export class InstallPlanPreview extends React.Component<InstallPlanPreviewProps,
               </thead>
               <tbody>
                 { steps.map((step, key) => <tr key={key}>
-                  <td>{step.resource.name}</td>
-                  <td>
-                    <ResourceIcon kind={step.resource.kind === ClusterServiceVersionModel.kind ? referenceForModel(ClusterServiceVersionModel) : step.resource.kind} />
-                    {step.resource.kind}
-                  </td>
-                  <td>{step.resource.group}/{step.resource.version}</td>
-                  <td>
-                    {stepStatus(step.status)}
+                  <td>{ ['Present', 'Created'].includes(step.status)
+                    ? <ResourceLink
+                      kind={referenceForStepResource(step.resource)}
+                      namespace={obj.metadata.namespace}
+                      name={step.resource.name}
+                      title={step.resource.name} />
+                    : <React.Fragment>
+                      <ResourceIcon kind={referenceForStepResource(step.resource)} />
+                      <a onClick={() => installPlanPreviewModal({stepResource: step.resource})} style={{cursor: 'pointer'}}>{step.resource.name}</a>
```

Comment:
This link is not accessible to keyboard or screen readers since it doesn't have an `href`. Needing to set `cursor: 'pointer'` usually indicates an accessibility problem.

Does `<button className="btn btn-link">` work?